doc/ProductSearch.md

```diff
@@ -21,7 +21,7 @@ With the release of ViSenze's Catalog system, ViSearch Android SDK will now incl
 2. [Initialization](#2-initialization)
 3. [Solution APIs](#3-solution-apis)
      - 3.1 [Search by Image](#31-search-by-image)
-     - 3.2 [Search by ID](#32-search-by-id)
+     - 3.2 [Search by ID/recommendation](#32-search-by-id)
```

Comment:
change this to Recommendations

Reply:
update the path to `#32-recommendations`

Reply:
changed